CONTRIBUTING.md

```diff
@@ -1,15 +1,75 @@
+
+There may be a little voice inside your head that is telling you that
+you're not ready to be an open-source contributor; that your skills
+aren't nearly good enough to contribute. What could you possibly offer a
+project like this one?
+
+We assure you - the little voice in your head is wrong. If you can
+write code at all, you can contribute code to open-source. Contributing
+to open-source projects is a fantastic way to advance one's coding
+skills. Writing perfect code isn't the measure of a good developer (that
+would disqualify all of us!); it's trying to create something, making
+mistakes, and learning from those mistakes. That's how we all improve,
+and we are happy to help others learn.
+
+Being an open-source contributor doesn't just mean writing code, either.
+You can help out by writing documentation, tests, or even giving
+feedback about the project (and yes - that includes giving feedback
+about the contribution process). Some of these contributions may be the
+most valuable to the project as a whole, because you're coming to the
+project with fresh eyes, so you can see the errors and assumptions that
+seasoned contributors have glossed over.
+
+## Driving principles
+
+*fMRIPrep* is built around three overarching principles:
+
+1. **Robustness** - The pipeline adapts the preprocessing steps depending on
+   the input dataset and should provide results as good as possible
+   independently of scanner make, scanning parameters or presence of additional
+   correction scans (such as fieldmaps).
+1. **Ease of use** - Thanks to dependence on the BIDS standard, manual
+   parameter input is reduced to a minimum, allowing the pipeline to run in an
+   automatic fashion.
+1. **"Glass box"** philosophy - Automation should not mean that one should not
+   visually inspect the results or understand the methods.
+   Thus, *fMRIPrep* provides visual reports for each subject, detailing the
+   accuracy of the most important processing steps.
+   This, combined with the documentation, can help researchers to understand
+   the process and decide which subjects should be kept for the group level
+   analysis.
+
+These principles distill some design and organizational foundations:
+
+  1. The tool only and fully supports BIDS and BIDS-Derivatives for the input and output data.
+  1. The tool is packaged as a fully-compliant [BIDS-App][link_bidsapps], not just in its user
+     interface, but also in the continuous integration, testing and delivery.
+  1. The tool is rigorously restricted to functional MRI preprocessing, including (but not limited to):
+     input/output metadata assessment, head-motion correction, susceptibility-distortion correction,
+     co-registration with anatomical data, spatial normalization to neuroimaging templates, 
+     noise components and other confounds extraction, etc.
+     In other words, the tool does not deal with time filtering, smoothing, modeling,
```

Comment:
However, the ICA-AROMA outputs are smoothed (at least currently). Should there be an asterisk here for that detail?

Reply:
Sure! Please send your suggestion via PR to my branch :)